UI/window-basic-settings.cpp

```diff
@@ -2118,20 +2108,11 @@ void OBSBasicSettings::LoadAdvOutputStreamingSettings()
 	ui->advOutMultiTrack5->setChecked(audioMixes & (1 << 4));
 	ui->advOutMultiTrack6->setChecked(audioMixes & (1 << 5));
 
-	bool is_multitrack_output = false;
 	obs_service_t *service_obj = main->GetService();
 	const char *protocol = nullptr;
 	protocol = obs_service_get_protocol(service_obj);
 	if (protocol) {
-		is_multitrack_output = allowsMultiTrack(protocol);
-	}
-
-	if (is_multitrack_output) {
-		ui->advStreamTrackWidget->setCurrentWidget(
-			ui->streamMultiTracks);
-	} else {
-		ui->advStreamTrackWidget->setCurrentWidget(
-			ui->streamSingleTracks);
+		SwapMultiTrack(QT_UTF8(protocol));
```

Comment:
We check `protocol` here and also check it inside the `SwapMultiTrack` function. Do we need to check it in both places?

Reply:
nope we don't; fixed in the next force push